src/topology/algebra/infinite_sum.lean

```diff
@@ -35,9 +35,10 @@ section has_sum
 variables [add_comm_monoid α] [topological_space α]
 
 /-- Infinite sum on a topological monoid
-The `at_top` filter on `finset α` is the limit of all finite sets towards the entire type. So we sum
-up bigger and bigger sets. This sum operation is still invariant under reordering, and a absolute
-sum operator.
+The `at_top` filter on `finset β` is the limit of all finite sets towards the entire type. So we sum
```

Comment:
The line above and this one are currently [running together in the docs](https://leanprover-community.github.io/mathlib_docs/topology/algebra/infinite_sum.html#has_sum)
```suggestion

The `at_top` filter on `finset β` is the limit of all finite sets towards the entire type. So we sum
```